app/models/bulk_tagging/tag_mapping.rb

```diff
@@ -24,7 +24,7 @@ class TagMapping < ActiveRecord::Base
 
     def content_id
       @content_id ||=
-        Services.publishing_api.lookup_content_id(base_path: content_base_path)
+        Services.publishing_api.lookup_content_id(base_path: content_base_path, exclude_unpublishing_types: %w[vanish gone], exclude_document_types: ['gone'])
```

Comment:
I'd consider just making both empty arrays, as it might be good to just not care about the states. But I'm fine with this also.